src/korean.js

```diff
@@ -129,8 +157,17 @@ class Korean {
   }
 
   doPast (word) {
-    console.info(word);
-    // stuff for past tense
+    const presentTense = this.doPresent(word);
```

Comment:
@minsooshin Fasho, my mistake on getting the doPresent changes confused with doPast.